pkg/configuration/configuration.go

```diff
@@ -97,7 +102,8 @@ type TestConfiguration struct {
 	AcceptedKernelTaints []AcceptedKernelTaintsInfo `yaml:"acceptedKernelTaints,omitempty" json:"acceptedKernelTaints,omitempty"`
 	SkipHelmChartList    []SkipHelmChartList        `yaml:"skipHelmChartList" json:"skipHelmChartList"`
 	// CheckDiscoveredContainerCertificationStatus controls whether the container certification test will validate images used by autodiscovered containers, in addition to the configured image list
-	CheckDiscoveredContainerCertificationStatus bool `yaml:"checkDiscoveredContainerCertificationStatus" json:"checkDiscoveredContainerCertificationStatus"`
+	CheckDiscoveredContainerCertificationStatus bool           `yaml:"checkDiscoveredContainerCertificationStatus" json:"checkDiscoveredContainerCertificationStatus"`
+	ProtocolName                                []ProtocolName `yaml:"validProtocolName" json:"validProtocolName"`
```

Comment:
Same here []string is enough here:
ProtocolName                                []ProtocolName 
-> 
validProtocolNames []string